stable/mongodb-replicaset/templates/_helpers.tpl

```diff
@@ -61,3 +61,19 @@ Create the name for the key secret.
         {{- template "mongodb-replicaset.fullname" . -}}-keyfile
     {{- end -}}
 {{- end -}}
+
+{{/*
+Create the name for the client full name.
+*/}}
+{{- define "mongodb-replicaset.truncForSuffix" -}}
```

Comment:
Instead of adding another template, we should just adjast the fullname template. This will guarantee consistent naming throughout the chart.

Reply:
thats fair